### DISCUSSION TITLE
Have problem with notification extension

### DISCUSSION THREAD

**Discussion by Dmitriy-Holovnia** [codemagic-ci-cd/codemagic-docs#1845] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

iOS (Swift, Objective-C)

### Steps to reproduce

Cannot build the app. 

### Expected results

Build the app.

### Actual results

  Run script build phase 'Run Script' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'olympusRelaxPalace' from project 'olympusRelaxPalace')

❌  /Users/builder/clone/olympusRelaxPalace.xcodeproj: error: "OneSignalNotificationExtension" requires a provisioning profile with the App Groups feature. Select a provisioning profile in the Signing & Capabilities editor. (in target 'OneSignalNotificationExtension' from project 'olympusRelaxPalace')

[20:15:23] ERROR > Failed to archive /Users/builder/clone/olympusRelaxPalace.xcworkspace

### Build id (optional)

64a32c1b58984bd284fd00c4

**Comment by himesh-cm** [comment]
Duplicate #1848

**Reply by haitue15122021** [reply]
You can try use manual signing instead of automatic signing.
ceate and upload provisioning profile for both app and OneSignalNotificationExtension
